autoload/ag.vim

```diff
@@ -94,7 +94,7 @@ function! ag#Ag(cmd, args)
     let &grepformat=g:ag_format
     set t_ti=
     set t_te=
-    silent! execute a:cmd . " " . escape(l:grepargs, '|')
+    silent! execute a:cmd . " .\\{0,50}" . escape(l:grepargs, '|').".\\{0,50}"
```

Comment:
This breaks some searches. For instance, if I run `:Ag set` in my dotfiles, for some reason it doesn't return any matches. However if I run `:Ag -u set` in my dotfiles, I got lots of matches (from ignored files). I don't see any matches with `:Ag -u set` when using this branch.